pommerman/network/ion_server/network.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+"""IonServer Network handler - This contains functions responsible Server-Client communication (You shouldn't use this file directly)"""
+
+# Imports
+from asyncio import new_event_loop, set_event_loop, get_event_loop
+from websockets import serve
+from threading import Thread
+from .constants import SPCommands as spc
+from .constants import NetworkCommands as nic
+from time import time, sleep
+from os import getcwd, path
+from re import fullmatch
+from gzip import compress
+from rapidjson import loads, dumps, load
+from json import JSONDecodeError
+from uuid import uuid4
```

Comment:
It's much clearer in the code to do:
```
import os
...
<os.path>
```

than it is to do:

```
from os import path
...
<path>
```

The latter also might get overwritten locally (or globally!) by a function that very reasonably assigns path.

Consequently, please change all of these to import the parent and then call that parent below.

Reply:
Done. (I only sort-of agree with you on this) (It took 30m+ to do and my hands are numb now)